packages/x-date-pickers-pro/src/internal/hooks/validation/useDateRangeValidation.ts

```diff
@@ -51,6 +46,11 @@ export const validateDateRange: Validator<
     return dateValidations;
   }
 
+  // for partial input
+  if (start === null || end === null) {
```

Comment:
From what I see, `isRangeValid` will return false anyway if one of the two date is null.
Are you sure this change works how you want it to work ?

Reply:
`isRangeValid` is not applied if one of the date is `null` because of the early return.